### DISCUSSION TITLE
cropper-container and cropper-canvas same size

### DISCUSSION THREAD

**Discussion by KeldJ** [CropperBlazor/Cropper.Blazor#202] [Q&A] [answered]
Is there anyway to have the cropper-container to have the same size as the cropper-canvas(image).

In the app were working on were uploading/manipulating images in a dialog(MudBlazor) and the uploaded image will allways be 100% width(- margin) of the dialog. The prevent cropping outside the image i would like the image and the container to have the same size.

**Comment by MaxymGorn** [comment, marked as answer]
Hi @KeldJ. You can use difference View Mode for cropper component in options cropper: 
<img width="1079" alt="image" src="https://github.com/CropperBlazor/Cropper.Blazor/assets/50423072/5637e6a5-3961-4080-825a-b0a6379f8059">
Demo site: https://cropperblazor.github.io/demo
I guess "VM3" suits you!
